src/ansys/tools/visualization_interface/backends/pyvista/pyvista.py

```diff
@@ -115,8 +115,8 @@ def __init__(
         # PyVista plotter
         self._pl = None
 
-        # List of picked objects in MeshObject format.
-        self._picked_list = set()
+        # Dictionary of picked objects in MeshObject format.
+        self._picked_list = {}
```

Comment:
Could you change the name of the variable to dict?

```suggestion
        self._picked_dict = {}
```

Reply:
Sounds like a great idea